recipes/xxmotif/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "1.6" %}
+
+package:
+  name: xxmotif
+  version: {{ version }}
+
+source:
+  fn: xxmotif-{{ version }}.tar.gz
+  url: https://github.com/soedinglab/xxmotif/archive/{{ version }}.tar.gz
+  sha256: 235eb8ad562c4757a549751bb290f443ff70c29a142edbff0f03440f097b61cb
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - gcc
```

Comment:
Unfortunately this code is very old and fragile. For reasons that elude me it apparently uses private gcc implementation details. I haven't managed to build it with llvm yet and I would rather not touch the code if I can avoid.